w1r3/infra/metrics/main.tf

```diff
@@ -33,6 +33,12 @@ locals {
       display_name = "Memory Usage"
       unit         = "1{memory}"
     }
+    rss = {
+      metric-prefix = "ssb/w1r3/ps/mem"
```

Comment:
I'm not certain this will do what I'm guess it will do. My intention is it will override the default from https://github.com/googleapis/storage-shared-benchmarking/pull/203/files#diff-44711931e6ef3cfe086149e3d8a4575b421669b89d01595058345d98d98a832dR15-R17

Reply:
No, you need to do something around like 50 too.  Maybe:

```
type = "workload.googleapis.com/lookup(each.value, "metric-prefix", ${var.metric-prefix})/${var.metric-prefix}/${each.key}"
```

But frankly I have not used `lookup()` in an interpolation, so I would test it if I was in your shoes.

Reply:
I ended up introducing a `type_name` property to be used as the name rather than the key. Makes the use of the declarations easier.